src/main/java/com/cloudbees/jenkins/GitHubSetCommitStatusBuilder.java

```diff
@@ -29,7 +29,7 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         for (GitHubRepositoryName name : GitHubRepositoryNameContributor.parseAssociatedNames(build.getProject())) {
             for (GHRepository repository : name.resolve()) {
                 listener.getLogger().println(Messages.GitHubCommitNotifier_SettingCommitStatus(repository.getUrl() + "/commit/" + sha1));
-                repository.createCommitStatus(sha1, GHCommitState.PENDING, build.getAbsoluteUrl(), Messages.CommitNotifier_Pending(build.getDisplayName()));
+                repository.createCommitStatus(sha1, GHCommitState.PENDING, build.getAbsoluteUrl(), Messages.CommitNotifier_Pending(build.getProject().getFullName()), build.getProject().getFullName());
```

Comment:
Any reasons to change the <code>DisplayName</code> to the project's full name. In such cases the status info will lose the information about the build.